src/Entity/Adherent.php

```diff
@@ -536,6 +568,13 @@ class Adherent implements UserInterface, UserEntityInterface, GeoPointInterface,
      */
     private $receivedDelegatedAccesses;
 
+    /**
+     * @var AdherentCommitment
+     *
+     * @ORM\OneToOne(targetEntity="App\Entity\AdherentCommitment", mappedBy="adherent", cascade={"all"}, fetch="EXTRA_LAZY")
```

Comment:
`EXTRA_LAZY` has no sense here.
Inverse sides of one-to-one associations can not be lazy, technically. There is no foreign key on the inverse side, hence it is impossible to decide whether to proxy it or not.